extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1523,21 +1523,15 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Renders the open attachment UI affordance.
+   * Installs the page attachment extension.
    * @private
    */
-  renderOpenAttachmentUI_() {
-    // AttachmentEl can be any component that extends draggable drawer.
-    const attachmentEl = this.element.querySelector(
-      'amp-story-page-attachment, amp-story-page-outlink, amp-story-shopping-attachment'
+  installPageAttachmentExtension_() {
+    const elementsThatRequireExtension = this.element.querySelector(
+      'amp-story-page-attachment, amp-story-page-outlink'
```

Comment:
This makes it so the extension loading went from

`amp-story.js` -> `amp-story-page-attachment` + `amp-story-shopping` -> shopping config load -> render

to 

`amp-story.js` -> `amp-story-shopping` -> `amp-story-page-attachment` + shopping config load -> render

What's the performance impact of this change?

Reply:
It's to prevent the extension from installing if there are no `amp-story-shopping-tag` elements on the page. 
That's an edge case that happens if the experience is misconfigured.
I'm not sure we need this check since we communicate that tags are needed to display the experience.
Reverted.